scss/_patterns_form-tick-elements.scss

```diff
@@ -54,6 +54,19 @@
     }
   }
 
+  %vf-pseudo-tick-box-checked-hover {
+    @extend %vf-pseudo-tick-box-checked;
```

Comment:
Is this needed? Won't changing only the background color work?

Reply:
Mmh I need to double check but I thought that since only `%vf-pseudo-tick-box-checked-hover` was extended for the checked and hover state I needed to extend the checked state